src/Illuminate/Validation/Rule.php

```diff
@@ -35,22 +35,46 @@ public static function exists($table, $column = 'NULL')
      * Get an in constraint builder instance.
      *
      * @param  array  $values
+     * @param  bool  $array_keys
      * @return \Illuminate\Validation\Rules\In
      */
-    public static function in(array $values)
+    public static function in(array $values, bool $array_keys = false)
```

Comment:
These default values should be changed to null, this will fix the tests... Also we don't use snake case for variable names, you could think about other name for this or just convert it for camel case...